Library/Formula/gtk+.rb

```diff
@@ -14,8 +15,8 @@ class Gtkx < Formula
   depends_on "jasper" => :optional
   depends_on "atk"
   depends_on "pango"
-  depends_on :x11 => ["2.3.6", :recommended]
   depends_on "gobject-introspection"
+  option "with-quartz-relocation"
```

Comment:
Let's move the option above the dependencies, with space between the two blocks.